docs/src/content/docs/hi/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,289 @@
+---
+title: CSS और स्टाइलिंग
+description: जानें कि अपनी Starlight साइट को कस्टम CSS के साथ कैसे शैली करें या Tailwind CSS के साथ एकीकृत करें।
+---
+
+आप अपनी Starlight साइट को कस्टम CSS फ़ाइलों के साथ स्टाइल कर सकते हैं या Starlight Tailwind प्लगइन का उपयोग कर सकते हैं।
+
+## कस्टम CSS शैलियाँ
+
+Starlight की डिफ़ॉल्ट शैलियों को संशोधित या विस्तारित करने के लिए अतिरिक्त CSS फ़ाइलें प्रदान करके अपनी Starlight साइट पर लागू शैलियों को अनुकूलित करें।
+
+1. अपनी src/ निर्देशिका में एक CSS फ़ाइल जोड़ें।
+   उदाहरण के लिए, आप पेज शीर्षकों के लिए व्यापक डिफ़ॉल्ट कॉलम चौड़ाई और बड़ा टेक्स्ट आकार सेट कर सकते हैं:
+
+   ```css
+   /* src/styles/custom.css */
+   :root {
+   	--sl-content-width: 50rem;
+   	--sl-text-5xl: 3.5rem;
+   }
+   ```
+
+2. अपनी CSS फ़ाइल का पथ `astro.config.mjs` में Starlight के `customCss` सरणी में जोड़ें:
+
+   ```diff lang="js"
+   // astro.config.mjs
+   import { defineConfig } from 'astro/config';
+   import starlight from '@astrojs/starlight';
+
+   export default defineConfig({
+   	integrations: [
+   		starlight({
+   			title: 'Docs With Custom CSS',
+   			customCss: [
+   +				// आपकी कस्टम CSS फ़ाइल से संबंधित पथ
+   +				'./src/styles/custom.css',
+   			],
+   		}),
+   	],
+   });
+   ```
+
+आप Starlight द्वारा उपयोग किए गए सभी CSS कस्टम गुणों को देख सकते हैं जिन्हें आप [GitHub पर `props.css` फ़ाइल](https://github.com/withastro/starlight/blob/main/packages/starlight/style/props.css) में अपनी साइट को अनुकूलित करने के लिए सेट कर सकते हैं।
+
+## Tailwind CSS
+
+Astro परियोजनाओं में Tailwind CSS समर्थन [Astro Tailwind एकीकरण](https://docs.astro.build/hi/guides/integrations-guide/tailwind/) द्वारा प्रदान किया जाता है।
+Starlight की शैलियों के साथ संगतता के लिए Tailwind को कॉन्फ़िगर करने में सहायता के लिए Starlight एक पूरक Tailwind प्लगइन प्रदान करता है।
+
+Starlight प्लगइन निम्नलिखित कॉन्फ़िगरेशन लागू करता है:
+
+- Tailwind `डार्क` कॉन्फ़िगर करना: Starlight के डार्क मोड के साथ काम करने के लिए वेरिएंट।
```

Comment:
```suggestion
- Tailwind `dark:` कॉन्फ़िगर करना: Starlight के डार्क मोड के साथ काम करने के लिए वेरिएंट।
```

This should not be translated as this refers to the Tailwind [variant](https://tailwindcss.com/docs/dark-mode).